contracts/ReaperAutoCompound-Liquid-v2-Beethoven.sol

```diff
@@ -290,6 +280,11 @@ contract ReaperAutoCompound_LiquidV2_Beethoven is ReaperBaseStrategy {
 
         IMasterChefv2(MASTER_CHEF).withdrawAndHarvest(poolId, balanceOfPool(), address(this));
 
+        // should we convert rewardToken to bpToken?
+        // 
+        // retireStrat is called as part of vault.upgradeStrat(), which would
+        // only transfer bpTokens to the new strat and the reward tokens would
+        // be stuck in the vault.
```

Comment:
Considered doing this but I'm worried in case of low reward amount that the swaps would not be worth it or fail.
This way we can guarantee the transaction goes through and the owner of the vault can retrieve the tokens with inCaseTokensGetStuck()